src/cluster_migrateslots.c

```diff
@@ -353,6 +358,151 @@ void fireModuleSlotMigrationEvent(slotMigrationJob *job, int subevent) {
     zfree(info.slot_ranges);
 }
 
+/* Encode active slot migrations into a string for storage into an RDB aux
+ * field.
+ *
+ * Format is:
+ * <job_name>:<source_node_name>:<start slot>-<end slot>( <start-slot2>-<end slot2>...)
```

Comment:
Does the search module have access to this information and/or does it need to? While we're loading from the snapshot and building the index?

Reply:
Yeah - I was thinking that if you do load the RDB on an older version and had, for whatever reason, taken the snapshot while a migration in progress - it is probably better to let the downgrade node ignore the slot imports. It will still cleanup unowned slots.

Suffice to say - the RDB is still valid even if you are not aware of the slot imports. It is just that there will be some dirty keys in unowned slots